pkg/helm/controller.go

```diff
@@ -297,15 +297,15 @@ func serviceAccount(chart *helmv1.HelmChart) *core.ServiceAccount {
 func args(chart *helmv1.HelmChart) []string {
 	if chart.DeletionTimestamp != nil {
 		return []string{
-			"delete",
-			"--purge", chart.Name,
+			"uninstall",
```

Comment:
Does this mean a helm 2 chart won't delete now?

Reply:
no that was an old update that meant to only use v3, pushing the new update now

Reply:
@ibuildthecloud The new update will pass only delete/install and the script will handle the installation or deletion for helm v2 and v3